packages/preferences/src/browser/preferences-decorator.ts

```diff
@@ -83,4 +83,23 @@ export class PreferencesDecorator implements TreeDecorator {
         }
         return defaultValue;
     }
+
+    private buildTooltip(data: PreferenceDataProperty): String {
+        let tooltips: string = '';
+        if (data.description) {
+            tooltips = data.description;
+        }
+        if (data.defaultValue) {
+            tooltips += `\nDefault: ${JSON.stringify(data.defaultValue)}`;
+        } else if (data.default !== undefined) {
+            tooltips += `\nDefault: ${JSON.stringify(data.default)}`;
+        }
+        if (data.minimum) {
+            tooltips += `\nMin: ${data.minimum}`;
+        }
+        if (data.enum) {
+            tooltips += `\nAccepted Value(s): ${data.enum.join(', ')}`;
```

Comment:
[minor]: If you do not want to count the number of enum and insert the _plural_ `s` based on it; please use `Values` instead of `Value(s)`. Thanks!

Reply:
Done